src/form-elements/FormElementNumber.tsx

```diff
@@ -40,10 +40,17 @@ function FormElementNumber({
     [element, onChange],
   )
   const htmlInputElementRef = React.useRef<HTMLInputElement>(null)
-  const handleWheel = React.useCallback((e) => {
-    e.preventDefault()
+  const [trigger, setTrigger] = React.useState<boolean>(false)
+
+  const handleWheel = React.useCallback(() => {
+    htmlInputElementRef.current?.blur()
+    setTrigger((trigger) => !trigger)
   }, [])
 
+  React.useEffect(() => {
+    htmlInputElementRef.current?.focus({ preventScroll: true })
+  }, [trigger])
+
   React.useEffect(() => {
```

Comment:
Can we please remove this effect and pass `handleWheel` to the `onWheel` prop on the `<input />`